apps/condo/domains/billing/access/ValidateQRCodeService.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Generated by `createservice billing.ValidateQRCodeService --type mutations`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canValidateQRCode ({ authentication: { item: user } }) {
```

Comment:
This access function seems to be similar to `canPaymentByLink`. Can it be reused here?

Reply:
Yes, but I do not want to use access rules from the different domain